plugins/env/README.md

```diff
@@ -1,4 +1,39 @@
+## Env Plugin
 Plugin to manage ENV settings for projects and write .env files during deploy.
 
 Includes `/projects/:permalink/environment?deploy_group=permalink` endpoint that returns the .env content
 for a project and deploy_group.
+
+## GitHub to manage environment variables
+This plugin has an option to use GitHub repository for .env files.  Using GitHub provides better control
```

Comment:
`for .env files` -> `as source for environment variables`

Reply:
I'll make this change.